core/payments/payments.go

```diff
@@ -273,7 +273,7 @@ func (s *service) GetSubscriptionStatus(ctx context.Context, req *pb.RpcMembersh
 
 	// 5 - if requested any name has changed, then we need to update details of local identity
 	if isDiffRequestedName {
-		s.profileUpdater.UpdateGlobalNames()
+		s.profileUpdater.UpdateGlobalNames(status.RequestedAnyName)
```

Comment:
Can be empty for some tiers that feature NO any name, but I think it will work here...